### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by homes32** [mycargus/nacho-nacha#1]
Is there any documentation or sample code available on how to generate an ACH file using this library?

**Comment by mycargus** [comment]
Hello! No documentation yet as the library is incomplete. Here's the vision:

```
var fileHeader = FileHeaderRecord.new(...);
var fileControl = FileControlRecord.new(...);
var batches = new List<Batch>(...);
var nachaFile = NachaFile.new( fileHeader, batches, fileControl );

nachaFile.generateAchFile();
```

see https://github.com/mycargus/nacho-nacha/blob/master/Src/Nacha.Core/NachaFile.cs#L38

Further abstraction is possible and would be even better, of course. The ease of use comes with built-in record, batch, and file validation and helpful messaging.

I stopped working on this library for lack of community participation but would love to see it grow. Please feel free to submit a pull request.

**Comment by robson26** [comment]
Hello,

Good start on this library! I started building off of this for a business need at the time, but that project has since lost steam so I stopped.

I'm knowledgeable about the industry and may be able to contribute. 

I still can't believe there isn't a library available to create ACH files, even for purchase.

**Comment by homes32** [comment]
agreed robson26, its crazy that there isn't a good library out there. I also work in the industry and would be willing to help, but I'm not a programmer by trade (I just hack around a bit for fun or to fulfill a need) and OOP is pretty new to me so I would need guidance to contribute any major chunks of code. My end goal would be to have a small program that I could schedule to run and it would either grab the info from a database or munch it from a .csv and generate an ACH file to inport into our GL.

**Comment by 935main** [comment]
Do any of you happen to have updates on this project where you've actually gotten it to _do_ something?  It looks like the most recent version gets a good way there but no NACHA file is actually written, there's no writeAchFile() method or anything.

**Comment by mycargus** [comment]
@935main Please see my comment above. If you'd like to see more work done on this project, feel free to submit a pull request. :)
